### PR TITLE
GH-280: Polishing: Consumer Message Header, Docs

### PR DESCRIPTION
**Pull request by garyrussell**
See: https://github.com/spring-projects/spring-kafka/issues/280

Add consumer as a mapped header.

### CODE REVIEW

File: spring-kafka/src/test/java/org/springframework/kafka/kstream/KafkaStreamsTests.java [hunk 1]
```diff
@@ -80,8 +80,8 @@
 	@Autowired
 	private SettableListenableFuture<String> resultFuture;
 
-	@Autowired
-	private KafkaEmbedded kafkaEmbedded;
+//	@Autowired
+//	private KafkaEmbedded kafkaEmbedded;
```
Comment (artembilan) [inline spring-kafka/src/test/java/org/springframework/kafka/kstream/KafkaStreamsTests.java]: May I know the reason?
Thanks

Comment (garyrussell) [inline spring-kafka/src/test/java/org/springframework/kafka/kstream/KafkaStreamsTests.java]: The field is not used in the test (STS `unused` warning - and you know I hate those warnings 😄 )

We can remove it altogether; I left it as a comment so as to draw your attention during the review !

Comment (artembilan) [inline spring-kafka/src/test/java/org/springframework/kafka/kstream/KafkaStreamsTests.java]: Ha! Good point!
Now I recall that I did that to add a sample to the docs.
I'll polishing it during merge.
Thank you!

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-kafka/commit/a71054273cd3e9d0327fbf05da6bfff47b38e59b

**Pull request closed without merging**